src/software/jetson_nano/gpio.cpp

```diff
@@ -10,17 +10,17 @@
 
 #include "software/logger/logger.h"
 
-GPIO::GPIO(std::string gpio_number, GpioDirection direction, GpioState initial_state)
+Gpio::Gpio(std::string gpio_number, GpioDirection direction, GpioState initial_state)
 {
-    // Setup the provided GPIO pin
+    // Setup the provided Gpio pin
     gpio_number_ = gpio_number;
 
     auto export_gpio_fs = std::ofstream("/sys/class/gpio/export");
     export_gpio_fs << gpio_number;
     export_gpio_fs.close();
 
     auto set_direction_fs =
-        std::ofstream("/sys/class/gpio/gpio" + gpio_number + "/direction");
+        std::ofstream("/sys/class/Gpio/Gpio" + gpio_number + "/direction");
```

Comment:
this opens a file stream and it expects the directory to be /sys/class/gpio/gpio (should remain the same as before)